Files/Helpers/AdaptiveLayoutHelpers.cs

```diff
@@ -2,6 +2,7 @@
 using Files.ViewModels.Previews;
 using System;
 using System.Linq;
+using System.Threading.Tasks;
```

Comment:
Aaand this can go away too... this is the last thing, promise.

Reply:
You wouldn't be able to sleep well, eh? 😆

Reply:
Same 😅

Reply:
Fixed in https://github.com/files-community/Files/pull/3979/commits/cd49e36b2b2eb6ee7f5c152c7085ff0a29033bc5 :+1: @gave92